### ISSUE TITLE
Crashes AeroSpace v0.13.3-Beta e03bfac3328ede090d29e736f2cb0103574f1ac8

### ISSUE THREAD

**Issue by frankroeder** [nikitabobko/AeroSpace#349]
The latest beta release crashes when switching workspaces or changing the layout.

Version:
AeroSpace v0.13.3-Beta e03bfac3328ede090d29e736f2cb0103574f1ac8

Log:

```log
##### AeroSpace Runtime Error #####

Please report to:
    https://github.com/nikitabobko/AeroSpace/issues/new

Message: 
Version: 0.13.3-Beta
Git hash: e03bfac3328ede090d29e736f2cb0103574f1ac8
Coordinate: /Users/Nikita.Bobko/a/AeroSpace/Sources/AppBundle/tree/Workspace.swift:91:14 ==(_:_:)
recursionDetectorDuringFailure: false
cli: false

Stacktrace:
0   AeroSpace                           0x0000000100817fa0 $s6Common6errorT_4file4line6column8functionxSS_SSS2iSStlFs5NeverO_Tg5 + 784
1   AeroSpace                           0x0000000100835974 $s6Common5check__4file4line6column8functionySb_S2SS2iSStF + 56
2   AeroSpace                           0x0000000100658e38 $s9AppBundle14updateTrayTextyyFSSAA7Monitor_pXEfU1_ + 320
3   AeroSpace                           0x000000010065907c $sSlsE3mapySayqd__Gqd__7ElementQzKXEKlFSay9AppBundle7Monitor_pG_SSTg503$s9c30Bundle14updateTrayTextyyFSSAA7E8_pXEfU1_AGTf1cn_nTf4ng_n + 112
4   AeroSpace                           0x0000000100658b18 $s9AppBundle14updateTrayTextyyF + 184
5   AeroSpace                           0x00000001006add7c $s9AppBundle14refreshSession7startup10forceFocus4bodyxSb_SbxyXEtlFyt_Tg503$s9A43Bundle9MacWindowC14garbageCollectyyFyyXEfU_AA9WorkspaceCTf1nnnc_nTm + 1612
6   AeroSpace                           0x00000001006c7c3c $s9AppBundle9MacWindowC14garbageCollectyyF + 756
7   AeroSpace                           0x00000001006c8e88 $s9AppBundle12destroyedObs33_AFFA185B3FDC2EA3EF5AFA96A7308179LL_2ax5notif4dataySo13AXObserverRefa_So011AXUIElementP0aSo08CFStringP0aSvSgtFTo + 72
8   HIServices                          0x000000019e4e83b0 _XXMIGPostNotification + 896
9   HIServices                          0x000000019e51349c _XPostNotification + 360
10  HIServices                          0x000000019e4ca340 mshMIGPerform + 208
11  CoreFoundation                      0x00000001980b1a6c __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE1_PERFORM_FUNCTION__ + 60
12  CoreFoundation                      0x00000001980b198c __CFRunLoopDoSource1 + 524
13  CoreFoundation                      0x00000001980b0350 __CFRunLoopRun + 2244
14  CoreFoundation                      0x00000001980af434 CFRunLoopRunSpecific + 608
15  HIToolbox                           0x00000001a285919c RunCurrentEventLoopInMode + 292
16  HIToolbox                           0x00000001a2858fd8 ReceiveNextEventCommon + 648
17  HIToolbox                           0x00000001a2858d30 _BlockUntilNextEventMatchingListInModeWithFilter + 76
18  AppKit                              0x000000019b90ecc8 _DPSNextEvent + 660
19  AppKit                              0x000000019c1054d0 -[NSApplication(NSEventRouting) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 700
20  AppKit                              0x000000019b901ffc -[NSApplication run] + 476
21  AppKit                              0x000000019b8d9240 NSApplicationMain + 880
22  SwiftUI                             0x00000001c3aa63b8 OUTLINED_FUNCTION_12 + 13968
23  SwiftUI                             0x00000001c420fddc OUTLINED_FUNCTION_3 + 196
24  SwiftUI                             0x00000001c460ae0c OUTLINED_FUNCTION_0 + 140
25  AeroSpace                           0x0000000100653500 main + 36
26  dyld                                0x0000000197c47154 start + 2476
```

**Comment by nikitabobko** [comment]
Thanks for the report! Should be fixed in 0.13.4. Since the fix is not trivial, I'd love to hear the confirmation that it's fixed on your side

**Comment by frankroeder** [comment]
It seems to be resolved with your fix. Thank you very much!
